lib/ser/ializer/field.rb

```diff
@@ -20,6 +20,7 @@ def initialize(name, options, config, &block)
         @deser = options[:deser]
         @if_condition = options[:if]
         @model_class = options[:model_class]
+        @description = options[:desc] || options[:description]
```

Comment:
```suggestion
        @description = options[:desc]
```

Reply:
Can you also add a test for this functionality.